tools/lint-md.sh

```
@@ -0,0 +1,29 @@
+# but never fails CI; locally it lets you choose to fix or not.
+#
+# Usage:
+#   tools/lint-md.sh                    # lint all *.md per .markdownlint-cli2.yaml
+#   tools/lint-md.sh --fix              # auto-fix what's auto-fixable
+#   tools/lint-md.sh path/to/file.md    # lint a single file
+#
+# Companion: tools/lint-md.ps1 (PowerShell equivalent).
+
+set -uo pipefail
+
+cd "$(dirname "$0")/.."
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)